### PR TITLE
feat: Add script for form validation and other functionalities

### PR DESCRIPTION
**Pull request by robinkataria**

### CODE REVIEW

File: login-signup/css/styles.css [hunk 1]
```diff
@@ -72,7 +56,7 @@ html {
   align-items: center;
 }
 
-.sso_button-logo {
+.sso__button__logo {
```
Comment (sharvan-sharma) [inline login-signup/css/styles.css]: 1. `.sso` isn't a block, we can use `.sso_button` as block class name and `.sso_button__logo` as element class name.
2. Avoid multi-level hireachy in BEM naming convention

File: login-signup/pages/signup/index.js [hunk 2]
```diff
@@ -0,0 +1,101 @@
+(() => {
+  const validateName = (name) => {
+    const re = /^([a-zA-Z]{1,4}\.)?([ a-zA-Z]{4,20})$/;
```
Comment (sharvan-sharma) [inline login-signup/pages/signup/index.js]: Define it as constant as follows
`const NAME_REGEX = /^([a-zA-Z]{1,4}\.)?([ a-zA-Z]{4,20})$/;`

File: login-signup/css/styles.css (hunk 1 above)
Comment (robinkataria) [reply login-signup/css/styles.css]: Rectified

File: login-signup/pages/signup/index.js (hunk 2 above)
Comment (robinkataria) [reply login-signup/pages/signup/index.js]: Defined as instructed

**Pull request merged**